srcpkgs/WindowMaker/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'WindowMaker'
 pkgname=WindowMaker
-version=0.95.9
-revision=2
+version=0.96.0
+revision=0
```

Comment:
At Void, the revision starts at 1 and not at zero

Reply:
Ahh ok. I wasn't positive/sure what to make that. I will change it now. Thanks.